docs/grid-introducer.rst

```diff
@@ -0,0 +1,104 @@
+.. -*- coding: utf-8 -*-
+
+Grid Introducer
+===============
+
+This document explains use and operation of the "Grid Introducer" system.
+This system replaces the earlier "Introducer node" system.
+
+The "Grid Introducer" system is a means for client nodes to initially learn about available storage nodes.
+It also allows client nodes to receive updates to this information.
+The goal of this document is to explain how to *use* this system,
+either as a user of a Tahoe-LAFS client node or as the operator of a Tahoe-LAFS grid or storage node.
+
+
+Use As A Client
+---------------
+
+You want your Tahoe-LAFS client to be able to find storage servers to access.
+This means your client needs both current and future connection information for those storage servers.
+By configuring your client with one small static connection hint,
+the "Grid Introducer" system enables the client to find more information and future updates to that information.
+
+Before attempting this configuration,
+you should have the Grid Introducer *cap* and *furl* which you received from the operator of a Tahoe-LAFS storage grid which uses the Grid Introducer system.
+
+The Grid Introducer is configured in ``tahoe.cfg`` in the ``[client]`` section's ``grid-introducer.cap`` and ``grid-introducer.furl`` items.
+For example::
+
+  [client]
+  grid-introducer.cap = URI:DIR2-RO:4bnx...
+  grid-introducer.furl = pb://sokl...@192.168.69.247:44801/eqpw...
+
```

Comment:
With the `grid-introducer.` prefix, this really feels like maybe it wants to be its own section so it can be `[grid-introducer]` ... `cap = ` etc.

Reply:
Yea, could be.  I don't know if there are pros or cons to different kinds of organization.  I took the `.` approach based on some prior art though - `shares.happy`, `web.port`, `onion.local_port` - and because this is configuration for the storage client, not an entirely new thing like the helper or the sftp frontend (to give two examples of other things configured in non-`[client]` sections).

If this were json, yaml, or nix, I'd probably make the config be a representation of the python object

```
{ "client":
  { "grid-introducer":
    { "cap": "URI:..."
    , "furl": "pb://..."
    }
  }
}
```

but of course we only get one layer of structure from ConfigParser.